packages/suite/src/actions/suite/protocolActions.ts

```diff
@@ -62,6 +73,16 @@ export const handleProtocolRequest = (uri: string) => (dispatch: Dispatch, getSt
         if (wcUri) {
             dispatch(walletConnectActions.walletConnectPairThunk({ uri: wcUri }));
         }
+    } else if (uri?.startsWith(SUITE_ANCHOR_DEEPLINK_PREFIX)) {
+        const anchor = uri.replace(SUITE_ANCHOR_DEEPLINK_PREFIX, '');
+
+        if (Object.values(SettingsAnchor).includes(anchor as SettingsAnchorValue)) {
```

Comment:
Can'' resist shilling my utils to other devs :joy: 
One less as.. 
```suggestion
        if (isArrayMember(anchor, Object.values(SettingsAnchor))) {
```

Reply:
great :+1: